### PR TITLE
fix: (Platform) Render fdp-form-field with wrapper

### PR DESCRIPTION
**Pull request by dimamarksman**
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/5281
#### Please provide a brief summary of this pull request.
This PR allows rendering of nested `fdp-form-field` 
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by fkolar — APPROVED** [review]

**Review by sKudum — APPROVED** [review]

File: libs/platform/src/lib/components/form/form.spec.ts [hunk 1]
```diff
@@ -210,32 +210,39 @@ describe('Nested Form Groups', () => {
                     <fdp-form-field #lastName id="lastName" label="Last Name">
                         <fdp-input name="lastName" [formControl]="lastName.formControl"></fdp-input>
                     </fdp-form-field>
-                    <fdp-form-field #favoriteColor id="favoriteColor" label="Favorite Color">
-                        <fdp-input name="favoriteColor" [formControl]="favoriteColor.formControl"></fdp-input>
-                    </fdp-form-field>
+                    <fdp-from-field-group>
```
Comment (kavya-b) [inline libs/platform/src/lib/components/form/form.spec.ts:213]:
```suggestion
                    <fdp-form-field-group>
```

File: libs/platform/src/lib/components/form/form.spec.ts [hunk 2]
```diff
@@ -210,32 +210,39 @@ describe('Nested Form Groups', () => {
...
+                        <fdp-form-field #state id="state" label="State">
+                            <fdp-input name="state" [formControl]="state.formControl"></fdp-input>
                         </fdp-form-field>
                         <fdp-form-field #city id="city" label="City">
                             <fdp-input name="city" [formControl]="city.formControl"></fdp-input>
                         </fdp-form-field>
-                        <fdp-form-field #state id="state" label="State">
-                            <fdp-input name="state" [formControl]="state.formControl"></fdp-input>
-                        </fdp-form-field>
+                        <fdp-from-field-group>
```
Comment (kavya-b) [inline libs/platform/src/lib/components/form/form.spec.ts:226]:
```suggestion
                    <fdp-form-field-group>
```

Comment (dimamarksman) [reply libs/platform/src/lib/components/form/form.spec.ts]: Thanks! 
fixed

File: libs/platform/src/lib/components/form/form.spec.ts (hunk 1 above)
Comment (dimamarksman) [reply libs/platform/src/lib/components/form/form.spec.ts]: Thanks! 
fixed

**Review by kavya-b — APPROVED** [review]
LGTM!

**Pull request merged**